src/main/java/mate/academy/springboot/datajpa/mapper/impl/ProductMapper.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.springboot.datajpa.mapper.impl;
+
+import mate.academy.springboot.datajpa.dto.request.ProductRequestDto;
+import mate.academy.springboot.datajpa.dto.response.ProductResponseDto;
+import mate.academy.springboot.datajpa.mapper.RequestDtoMapper;
+import mate.academy.springboot.datajpa.mapper.ResponseDtoMapper;
+import mate.academy.springboot.datajpa.model.Product;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProductMapper implements RequestDtoMapper<ProductRequestDto, Product>,
+        ResponseDtoMapper<Product, ProductResponseDto> {
+
+    @Override
+    public Product mapToModel(ProductRequestDto dto) {
+        Product product = new Product();
+        product.setPrice(dto.getPrice());
+        product.setTitle(dto.getTitle());
```

Comment:
add setting a category (just id needed)
``` 
product.setCategory(new Category(dto.getCategoryId()));
```
you can add getting category by id on the service layer -  `ProductService.save()` method